lib/ansible/keyword_desc.yml

```diff
@@ -61,7 +61,7 @@ serial: Explicitly define how Ansible batches the execution of the current play
 strategy: Allows you to choose the strategy plugin to use for the play. See :ref:`strategy_plugins`.
 tags: Tags applied to the task or included tasks, this allows selecting subsets of tasks from the command line.
 tasks: Main list of tasks to execute in the play, they run after :term:`roles` and before :term:`post_tasks`.
-timeout: Time limit for the task to execute in, if exceeded Ansible will interrupt and fail the task.
+timeout: Time limit for the task action to execute in, if exceeded, Ansible will interrupt the process. Timeout does not affect templating or looping.
```

Comment:
'affect' seems to be the wrong word here, timeout does not account for/ include/cover  templating and looping